pubspec.yaml

```diff
@@ -1,11 +1,11 @@
 name: http_parser
-version: 3.1.2-dev
+version: 3.1.2
```

Comment:
There's no need to release this, since only test code was changed.

Reply:
Reverted to 3.1.2-dev.